### ISSUE TITLE
YOLO608

### ISSUE THREAD

**Issue by CGCooke** [experiencor/basic-yolo-keras#70]
Hi Experiencor,
I'm interested in seeing the difference between the YOLOv2  416x416 and the YOLOv2 608x608 models,
Is this just a matter of generating weights using YAD2K and changing the size and anchors in config.json?,
Thanks again, awesome job!

**Comment by experiencor** [comment]
@CGCooke Do you mean to use this repo to test the difference between the 2 models trained using Darknet code via YAD2K code?

P/S: you can use this repo to train 2 models (416 and 608) and evaluate then thereafter.

**Comment by CGCooke** [comment]
Thanks @experiencor,

I've previously adapted the code from commit e6aad92  on 10 Aug to use the 608 model, and I was interested in the added benefit from using imgaug & masking in the loss function.

Do I also need to make any changes to do multi-class detection? (Apart from adding the extra classes in config.json)

**Comment by experiencor** [comment]
You don't need to make any changes. The size of the network auto adjusts to the number of classes. Why don't you try the latest code? The old code has issue with multi-class detection in older version of Keras.

**Comment by CGCooke** [comment]
Thank you,
I'm trying the new code, still trying to identify why the e6aad92 commit performs better on my dataset than the current code.
